### PR TITLE
feat(icons): update icons

### PR DESCRIPTION
**Pull request by AxelPeter**
- Add new icon `truck`
- Update icons `copy` and `bin`
- Rename `print` to `printer`
- Rename `transfer` to `arrow-transfer`

Fix #UK-159

### CODE REVIEW

File: packages/base/core/src/fonts/oui-icons/oui-icons.html [hunk 1]
```diff
@@ -2392,6 +2401,15 @@ <h1>oui-icons</h1>
...
 
 
+        <div class="preview">
+            <span class="inner">
+                <span class=" oui-icon-truck"></span>
+            </span>
+            <br>
+            <span class='label'>truck</span>
+        </div>
+
```
Comment (FredericEspiau) [inline packages/base/core/src/fonts/oui-icons/oui-icons.html:2411]: Extra WS, maybe you could add a linter to format this automatically

File: packages/base/core/src/fonts/oui-icons/oui-icons.html [hunk 2]
```diff
@@ -2392,6 +2401,15 @@ <h1>oui-icons</h1>
         </div>
 
 
+        <div class="preview">
+            <span class="inner">
+                <span class=" oui-icon-truck"></span>
+            </span>
+            <br>
+            <span class='label'>truck</span>
```
Comment (FredericEspiau) [inline packages/base/core/src/fonts/oui-icons/oui-icons.html:2409]:
```suggestion
            <span class="label">truck</span>
```

File: packages/base/core/src/fonts/oui-icons/oui-icons.html (hunk 1 above)
Comment (AxelPeter) [reply packages/base/core/src/fonts/oui-icons/oui-icons.html:2411]: This file is generated by the icon-generator, like everything in the `fonts/oui-icons/` folder.
We keep it to see the result of the generated icons.

File: packages/base/core/src/fonts/oui-icons/oui-icons.html [hunk 3]
```diff
@@ -2392,6 +2401,15 @@ <h1>oui-icons</h1>
         </div>
 
 
+        <div class="preview">
+            <span class="inner">
+                <span class=" oui-icon-truck"></span>
```
[the remark this answers is not in the document]
Comment (AxelPeter) [reply packages/base/core/src/fonts/oui-icons/oui-icons.html:2406]: Same here

File: packages/base/core/src/fonts/oui-icons/oui-icons.html (hunk 2 above)
Comment (AxelPeter) [reply packages/base/core/src/fonts/oui-icons/oui-icons.html:2409]: Same here

File: packages/base/core/src/fonts/oui-icons/oui-icons.html (hunk 1 above)
Comment (FredericEspiau) [reply packages/base/core/src/fonts/oui-icons/oui-icons.html:2411]: Well we should teach this generator how to write `html` then 😄

Comment (AxelPeter) [reply packages/base/core/src/fonts/oui-icons/oui-icons.html:2411]: Do your magic ! https://github.com/Workshape/icon-font-generator/blob/master/template/html.hbs :)

**Pull request merged**